spec.md

```diff
@@ -1364,6 +1369,7 @@ For `200 OK` and `201 Created` response codes, the following fields are defined:
 | syslog_drain_url | string | A URL to which logs MUST be streamed. `"requires":["syslog_drain"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
 | route_service_url | string | A URL to which the Platform MUST proxy requests for the address sent with `bind_resource.route` in the request body. `"requires":["route_forwarding"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
 | volume_mounts | array of [VolumeMount](#volume-mount-object) objects | An array of configuration for remote storage devices to be mounted into an application container filesystem. `"requires":["volume_mount"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
+| endpoints | array of [Endpoint](#endpoint-object) objects | The network endpoints that the Application uses to connect to the Service Instance. The endpoints need not to be reachable and the host names need not to resolvable from outside the service network. |
```

Comment:
"need not to be" reads a little funny to me, but I'm not sure if I can think of a better phrase.

Also, s/the host names need not to resolvable/the host names need not be resolvable/

Reply:
"Endpoints and hostnames will not necessarily be accessible or resolvable from outside the service network."

Reply:
nvm, I see wording was updated

Reply:
There are a couple of minor typos in the new wording. Suggested fixes in bold below.

The network endpoints that the Application uses to connect to the Service Instance. All Service Instance endpoints that **are** relevant for the Application MUST be in this list, even if the endpoints are not reachable or host names are not resolvable **from** outside the service network.